tests/nn/data_parallel/test_sharded_ddp_pytorch_parity.py

```diff
@@ -290,341 +285,9 @@ def run_ddp_parity_two_optim(rank, world_size, backend, temp_file_name, reduce_b
 @pytest.mark.parametrize("reduce_buffer_size", [0, 2 ** 20])
 def test_ddp_parity_two_optim(reduce_buffer_size):
     world_size = 2
-    backend = dist.Backend.NCCL
     mp.spawn(
         run_ddp_parity_two_optim,
-        args=(world_size, backend, tempfile.mkstemp()[1], reduce_buffer_size),
-        nprocs=world_size,
-        join=True,
-    )
-
-
-def run_test_two_inputs(rank, world_size, backend, device, temp_file_name, reduce_buffer_size):
```

Comment:
all the test below were duplicates, probably a merge which failed on somebody. These are not ddp parity, they are test of sharded_ddp features and are handled in the other test file

Reply:
checked in in another PR